third_party/pg_duckdb_extensions.cmake

```diff
@@ -1,7 +1,7 @@
 duckdb_extension_load(json)
 duckdb_extension_load(icu)
 # This is relative to third_party/duckdb dir
-duckdb_extension_load(cached_httpfs
-    SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../cached_httpfs"
-    INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../cached_httpfs/include"
-)
\ No newline at end of file
+duckdb_extension_load(httpfs
```

Comment:
Why not using the public one? (like `json`/`icu`?)

Reply:
Doesn't `duckdb_extension_load(httpfs)` just do exactly that?

For example with json:
```
y=# select * from duckdb.raw_query($$select extension_name, loaded, installed, install_mode, install_path from duckdb_extensions() where extension_name = 'json';$$);
NOTICE:  result: extension_name	loaded	installed	install_mode	install_path
VARCHAR	BOOLEAN	BOOLEAN	VARCHAR	VARCHAR
[ Rows: 1]
json	true	true	STATICALLY_LINKED	(BUILT-IN)


 raw_query
-----------

(1 row)
```

Reply:
Ah sorry I thought JSON was also out of tree...

Reply:
Ok, this is it: https://github.com/duckdb/pg_duckdb/pull/644/commits/7579326fb8fb580e0e72fd6bb82d616cebc0d829